### DISCUSSION TITLE
cannot verify github education pack

### DISCUSSION THREAD

**Discussion by SayaAEP** [community/community#55979] [GitHub Education] [answered]
### Hi everyone,

hello there. idk why i'm cannot verify my student id and change my public profile to real irl name (but don't want my name revealed).
![image](https://github.com/community/community/assets/91467886/4f7f6ae9-d300-4fa1-b357-fcc0ea516fc3)
![image](https://github.com/community/community/assets/91467886/c16feb71-ba3f-43f4-a255-4c94df93fbeb)

i'm upload my student id from pc with obs camera but main webcam pc are bad.

### A suggestion...(optional)

codespaces?, vscode, learning machine

**Comment by samus-aran** [comment, marked as answer]
Hi @TukangM 👋 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

**Comment by SayaAEP** [comment]
thanks for the answer. i'll try another methods.
